cg/constants/orderforms.py

```diff
@@ -1,19 +1,11 @@
 from enum import StrEnum
 
 from cg.constants import ANALYSIS_SOURCES, METAGENOME_SOURCES
-from cg.models.orders.order import OrderType
 
 SEX_MAP = {"male": "M", "female": "F", "unknown": "unknown"}
 REV_SEX_MAP = {value: key for key, value in SEX_MAP.items()}
-CONTAINER_TYPES = ["Tube", "96 well plate"]
 SOURCE_TYPES = set().union(METAGENOME_SOURCES, ANALYSIS_SOURCES)
 
-CASE_PROJECT_TYPES = [
```

Comment:
The removal of these unused constants fixed a circular import error

Reply:
They seem misleading either way